src/EventListener/AddFormatListener.php

```diff
@@ -145,4 +151,16 @@ private function getNotAcceptableHttpException(string $accept, array $mimeTypes
             implode('", "', $mimeTypes)
         ));
     }
+
+    /**
+     * Retrieves the list of accepted format in the request depending on some parameters.
+     */
+    private function getRequestAcceptedFormats(Request $request): array
+    {
+        if (!$request->attributes->has('_api_endpoint_type')) {
```

Comment:
you can call `get` directly and store the result in a variable to prevent an extra function call. (get will return `null` if the attribute is not defined)